### ISSUE TITLE
Add v1.24.10-rancher4-1

### ISSUE THREAD

**Issue by kinarashah** [rancher/rancher#40668]
Current latest version `v1.24.10-rancher3-1` has RC build of kubelet and the node version shows as `v1.24.x-xxx-dirty`. 

Will need to add final release `v1.24.10-rancher4-1` that uses final release tags, and node version would be `v1.24.10`.

**Comment by sowmyav27** [comment]
Using k8s versions - `v1.24.10-rancher4-1`and `v1.25.6-rancher4-1`. 

- provision an RKE1 cluster. Ensure cluster comes up Active. System Project page workloads are up and Active
- Ensure the node version shows the right k8s version - `v1.24.10-rancher4-1`or `v1.25.6-rancher4-1`.

**Comment by kinarashah** [comment]
v1.24: 
https://github.com/rancher/kubernetes/releases/tag/v1.24.10
https://github.com/rancher/hyperkube/releases/tag/v1.24.10-rancher4

**Comment by kinarashah** [comment]
Available to test with Rancher v2.6-head after build passes https://drone-publish.rancher.io/rancher/rancher/9173.

**Comment by rishabhmsra** [comment]
**Validated on rancher v2.6-head(9b65d8b), KDM pointing to dev-v2.6**

- Provisioned RKE1 v1.24.10-rancher4-1, k8s cluster(1-cp, 1-e, 1-w). 
- Cluster came Active with all the pods up and running
- Validated node version :

**Using kubectl :**
```
> kubectl version
Server Version: version.Info{Major:"1", Minor:"24", GitVersion:"v1.24.10", GitCommit:"5c1d2d4295f9b4eb12bfbf6429fdf989f2ca8a02", GitTreeState:"clean", BuildDate:"2023-01-18T19:08:10Z", GoVersion:"go1.19.5", Compiler:"gc", Platform:"linux/amd64"}
```

```
> kubectl get nodes
NAME          STATUS   ROLES          AGE   VERSION
rishabh-cp1   Ready    controlplane   19m   v1.24.10
rishabh-e1    Ready    etcd           19m   v1.24.10
rishabh-w1    Ready    worker         17m   v1.24.10
```
**Kubelet logs also shows:**
```
I0227 10:55:37.391296    7334 server.go:399] "Kubelet version" kubeletVersion="v1.24.10"
```
**Node yaml :**
```
nodeInfo:
      kubeProxyVersion: v1.24.10
      kubeletVersion: v1.24.10
```

**On UI :** 

![UI-versions](https://user-images.githubusercontent.com/36376154/221549501-b3d735ff-28e1-40b9-9d8e-60e53fc5d93f.png)

**Result :** 
- Node version is correct, hence closing the issue.
